core/api/routes/health.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import APIRouter
+from api.models.schemas.health import HealthCheck, ServiceStatus
+from api.resources import strings
+from api.services.health_checker import get_state_store_status
+
+
+router = APIRouter()
+
+
+@router.get("", name="health:get-status-of-services")
```

Comment:
instead of prefix in `router.include_router(ping.router, tags=["ping"], prefix="/ping")` should this not be instead `@router.get("/ping", name="health:get-status-of-services")`

Now I have to refer to api.py to figure out what route really hits this one

Reply:
I changed this per your suggestion - but I guess it is a matter of where you want to know it -- I was following the suggestion from the real world example